src/officehours/settings.py

```diff
@@ -333,8 +334,10 @@ def skip_auth_callback_requests(record):
 BLUEJEANS_TELE_NUM = os.getenv('BLUEJEANS_TELE_NUM', '1.312.216.0325')
 BLUEJEANS_INTL_URL = os.getenv('BLUEJEANS_INTL_URL', 'https://www.bluejeans.com/premium-numbers')
 
-ENABLED_BACKENDS = {'inperson'}
-DEFAULT_BACKEND = "inperson"
+IMPLEMENTED_BACKEND = Literal['inperson', 'zoom', 'bluejeans']
```

Comment:
I tried to use this type elsewhere via `settings`, but it doesn't seem like `mypy` can do much with because of the `settings` lazy-loading? Not exactly sure how all that works.

Reply:
Yeah, I'm using `mypy` and I tried setting up `django-stubs`, but it wasn't immediately clear to me whether it was working how I wanted it to/if I had it configured correctly. I decided to move this specific type out of `settings` to somewhat circumvent this, but also because details about what is implemented was (or probably should be) elsewhere, i.e. in `backends`.